<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CoreUI</class>
 <widget class="QMainWindow" name="CoreUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenCV Face Recognition System - Core</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="realTimeCaptureGroupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>641</width>
      <height>581</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="aboutLabel">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>541</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>基于OpenCV的视频人脸、表情识别</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="chooseButton">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>90</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>请选择视频</string>
     </property>
    </widget>
    <widget class="QLabel" name="filenamelabel">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>90</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>尚未选择文件</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="faceRecognitionCheckBox">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>170</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>统计人脸识别结果</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="expressionRecognitionCheckBox">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>240</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>统计表情识别结果</string>
     </property>
    </widget>
    <widget class="QPushButton" name="startButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>320</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>开始识别</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
